### ISSUE TITLE
I can't write on the console (All The Mods 6 1.4.0)

### ISSUE THREAD

**Issue by StickySli** [AllTheMods/Server-Scripts#80]
I can't write on the server CMD window when the server is up and running. Using All The Mods 6 version 1.4.0. 
I try to write "stop" like I normally do with my vanilla MC server to save the world and what not, but with the script I can't do this.
I tried investigating if the parameter "nogui" was present and I can't find it anywhere! Could this be the problem?

**Comment by jeremiahwinsley** [comment]
Is this the first launch? You will need to close it and relaunch it before the console starts working.

**Comment by StickySli** [comment]
NO, it is not. The server is running completely fine up and running and my friends can join, but I can't close it safely nor can I type on the terminal.

**Comment by arialthesiren** [comment]
I'm seeing the same thing. I launched the server.bat and was able to log into the server via the client but I'm unable to use any commands in game or via the console / cmd. I've had to completely reinstall the server files just to be able to stop the server as the CMD disappears with the X button but doesn't stop the server from working. i'm running version 1.5.7

correction, this time closing the cmd, after a second install! has allowed me to type in cmd :)
